python/lsst/ts/watcher/rules/mtm2_out_closed_loop_control.py

```diff
@@ -0,0 +1,130 @@
+# This file is part of ts_watcher.
+#
+# Developed for Vera C. Rubin Observatory Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = ["MTM2OutClosedLoopControl"]
+
+import logging
+import types
+import typing
+
+from lsst.ts import salobj, watcher
+from lsst.ts.xml.enums.MTM2 import PowerSystemState, PowerType
+from lsst.ts.xml.enums.Watcher import AlarmSeverity
+
+from ..base_rule import AlarmSeverityReasonType, NoneNoReason
+from ..remote_info import RemoteInfo
+
+
+class MTM2OutClosedLoopControl(watcher.BaseRule):
+    """Monitor the main telescope M2 is out of the closed-loop control.
+
+    Parameters
+    ----------
+    config : `types.SimpleNamespace`
+        Rule configuration, as validated by the schema.
+    log : `logging.Logger` or None, optional
+        Parent logger. (the default is None)
+    """
+
+    def __init__(
+        self, config: types.SimpleNamespace, log: logging.Logger | None = None
+    ) -> None:
+
+        remote_name = "MTM2"
+        remote_info = RemoteInfo(
+            remote_name,
+            0,
+            callback_names=["evt_powerSystemState", "evt_forceBalanceSystemStatus"],
+        )
+        super().__init__(
+            config,
+            f"MTM2OutClosedLoopControl.{remote_name}",
```

Comment:
MTM2 is already in the name. Consider just "MTM2OutClosedLoopControl".

Reply:
Fixed in the commit: [e5e703b](https://github.com/lsst-ts/ts_watcher/pull/83/commits/e5e703b275b024b0fd34185ef4cae12882298a75). Thanks!